IFC4x3/Properties/2/2$vDyZ4jX80fnC8diCnxQ3/DocProperty.xml

```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="RiskName" UniqueId="57b105f7-196d-4742-a5c5-096a7c076c11" PrimaryDataType="IfcLabel" />
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="x2vDyZ4jX80fnC8diCnxQ3" UniqueId="bfe4df23-12d8-4802-9c4c-227b0cc7b683" />
```

Comment:
What happened to this one? Seems it lost its name.

Reply:
error in deletion from IfcDoc, corrected